### ISSUE TITLE
Make sdss_access return compressed/uncompressed version of files

### ISSUE THREAD

**Issue by albireox** [sdss/marvin#298]
It would be good if Marvin could find both the compressed (`.fits.gz`) and uncompressed (`.fits`) version of any file. I think I natural place to implement that would be in `sdss_access`. Maybe we could add something to the paths in `tree` indicating that a part of the path is optional, e.g.

```
mangacube = $MANGA_SPECTRO_REDUX/{drpver}/{plate}/stack/manga-{plate}-{ifu}-LOGCUBE.fits(.gz)
```

`Path().full()` could check if one of the two versions actually exists on disk and return it. Otherwise it would returns the current version of the path (the one with .gz). I'm not sure if this would cause other problems down the line.

Alternatively, we could create a small wrapper in Marvin for `Path()`. All our requests to `Path()` should go through that wrapper that would have the behaviour above.

**Comment by havok2063** [comment]
Would you want `sdss_access` to also uncompress files upon rsync-ing them?  

The keyword arguments in `tree` are required ones.  I'm not sure we can set optional ones with a lot of hassle.  It might be better to leave the path definitions as is and define them as their original outputted selves, and just add an option in `Path` that would lop off any `.gz` extension and look for the uncompressed version.   And vice versa.  

Another option might be to just remove the `.gz` extension in all the `tree` path definitions and have `sdss_access` look for uncompressed files by default and fallback to compressed versions.  We might be able to just add a `.fits*` into the `Path` glob.   

Most of the methods in `Path` and `RsyncAccess` use `Path().full` so this could either make it easy adopt or cause problems in other functionality.  

In any case, I'd be worried about `sdss_access` finding and returning a mix of compressed and uncompressed versions that could mess up various other codes, or peoples' scripts.

Let's ping @joelbrownstein and get his opinions on this.

**Comment by albireox** [comment]
Sounds good. If this causes too many problems with Path().full() maybe we should just do it ourselves in Marvin. Any of those options to tell Path to look for a compressed/uncompressed file works for me. Regarding uncompressing the file on download, I think it shouldn't be the default, but maybe an option we pass to RsyncAccess.

**Comment by havok2063** [comment]
This was implemented in `sdss-access 1.0`
